lighthouse-core/gather/driver.js

```diff
@@ -1477,6 +1492,33 @@ class Driver {
     }
   }
 
+  /**
+   * @param {string} url
+   * @param {(LH.IcuMessage | string)[]} LighthouseRunWarnings
+   */
+  async getImportantStorageWarning(url, LighthouseRunWarnings) {
+    const usageData = await this.sendCommand('Storage.getUsageAndQuota', {
+      origin: url,
+    });
+    /** @type {Record<string, string>} */
+    const storageTypeNames = {
+      local_storage: 'Local Storage',
+      indexeddb: 'IndexedDB',
+      websql: 'Web SQL',
+    };
+    const locations = usageData.usageBreakdown
+      .filter(usage => usage.usage)
+      .map(usage => storageTypeNames[usage.storageType] || '')
+      .filter(Boolean);
+    if (locations.length) {
+      // TODO: Use Intl.ListFormat with Node 12
```

Comment:
can you file an issue for `Intl.ListFormat` too (and mention the case in multi-check-audit), otherwise this is going to sit here forever unfixed (see also: the other TODOs still in this file :)